src/bin/text/dune

```diff
@@ -28,6 +28,29 @@
   )
 )
 
+(executable
+  (name         Worker)
```

Comment:
Do you think you could just use lowercase everywhere ? Otherwise [terrible things](https://github.com/BinaryAnalysisPlatform/core-lwt/pull/5) are going to happen... :p (My link is juste about using `_` or `-` but same goes for lower/uppercase.

Reply:
Yep it's still wip, i'll clean this at the end

Reply:
OK. :)